Admin/FieldDescription.php

```diff
@@ -115,4 +116,29 @@ public function setParentAssociationMappings(array $parentAssociationMappings)
 
         $this->parentAssociationMappings = $parentAssociationMappings;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    final public function describesSingleValuedAssociation()
+    {
+        return in_array($this->mappingType, array(
+            ClassMetadata::MANY_TO_ONE,
+            'child',
```

Comment:
actually not sure if this is not ONE_TO_ONE. a child can only have one parent, and a parent can only have a particular child once.